test/test_sockets.py

```diff
@@ -349,3 +349,11 @@ def test_posix_proxy_sockets(self):
       with PythonTcpEchoServerProcess('7777'):
         # Build and run the TCP echo client program with Emscripten
         self.btest_exit('websocket/tcp_echo_client.c', args=['-lwebsocket', '-sPROXY_POSIX_SOCKETS', '-pthread', '-sPROXY_TO_PTHREAD'])
+
+
+class sockets64(sockets):
```

Comment:
Is this used anywhere?

Reply:
If you mean, is it run by CI anywhere, no, I just rand it locally.